src/main/java/org/graylog/integrations/dataadapters/GreyNoiseQuickIPDataAdapter.java

```diff
@@ -120,22 +125,22 @@ static LookupResult parseResponse(Response response) {
             } catch (JSONException | IOException e) {
                 LOG.error("An error occurred while parsing Lookup result [{}]", e.toString());
             }
-            return LookupResult.withoutTTL().multiValue(map)
-                               .build();
-        } else
+            return LookupResult.withoutTTL().multiValue(map).build();
+        } else {
             return LookupResult.empty();
+        }
```

Comment:
This must've been updated with the auto-format on save according to the Graylog formatting profile

Reply:
> @kingzacko1 Are you able to see the Integrations project on the Organization tab for this PR? I see it on my recent tab, but not the organization tab.

Checked it out this morning and it was there and available on the Organizations tab... but it was nowhere to be found yesterday - recent nor organization